OracleWebLogic/samples/12213-patch-wls-for-k8s/Dockerfile.patch-ontop-12213

```diff
@@ -34,18 +34,43 @@ FROM oracle/weblogic:12.2.1.3-developer
 # ----------
 MAINTAINER Monica Riccelli <monica.riccelli@oracle.com>
 
-# Environment variables required for this build (do NOT change)
+# Environment variables required for this build.
 # -------------------------------------------------------------
-# NOTE: This patch (p27117282 is needed only if the WebLogic binary image
-# is created manually from this Github project
+# NOTE: That patch (p27117282) is needed only if the WebLogic binary image
+# is created manually from this Github project under `dockerfiles/12.2.1.3`.
+# Comment out the ENV command if not needed.
 ENV PATCH_PKG1="p27117282_122130_Generic.zip"
 ENV PATCH_PKG2="p29135930_122130_Generic.zip"
 
-# Copy supplemental package and scripts
+# PATCH_PKG1 (27117282) is only needed if the WebLogic binary image
+# is created manually from this Github project under dockerfiles/12.2.1.3.
+# Uncomment the command to `Copy Patch 29135930` and comment the command 
```

Comment:
Shall we say "Otherwise, uncomment the command....", or " In other use cases, uncomment the command"?